botocore/docs/example.py

```diff
@@ -10,11 +10,18 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
-from botocore.docs.utils import traverse_and_document_shape
+from botocore.docs.shape import ShapeDocumenter
 from botocore.docs.utils import py_default
 
 
-class BaseExampleDocumenter(object):
+class BaseExampleDocumenter(ShapeDocumenter):
+    EVENT_NAME = ''
+
+    def __init__(self, service_name, operation_name, event_emitter):
```

Comment:
Just curious, is there any reason not to move the `__init__` into the base class?  Both this class and the other subclass of `ShapeDocumenter` have the same `__init__`.